diff --git a/dist/index.mjs b/dist/index.mjs
index 9823f2b9aca4f674b7d1f471db27587131c68468..f13522be8a11fc067dc77e2a185b9d31b7a10284 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -388,18 +388,18 @@ var ADBAutoInstaller = class {
 	async install() {
 		if (await this.check()) return;
 		await fs.ensureDir(this.installPath);
-		if (isMacOS) {
-			const { installADB } = await import("@autoglm.js/platform-tools-darwin");
-			await installADB(this.installPath);
-		}
-		if (isLinux) {
-			const { installADB } = await import("@autoglm.js/platform-tools-linux");
-			await installADB(this.installPath);
-		}
-		if (isWindows) {
-			const { installADB } = await import("@autoglm.js/platform-tools-windows");
-			await installADB(this.installPath);
-		}
+		// if (isMacOS) {
+		// 	const { installADB } = await import("@autoglm.js/platform-tools-darwin");
+		// 	await installADB(this.installPath);
+		// }
+		// if (isLinux) {
+		// 	const { installADB } = await import("@autoglm.js/platform-tools-linux");
+		// 	await installADB(this.installPath);
+		// }
+		// if (isWindows) {
+		// 	const { installADB } = await import("@autoglm.js/platform-tools-windows");
+		// 	await installADB(this.installPath);
+		// }
 	}
 	async check() {
 		return await fs.pathExists(this.adbPath);
